python_modules/dagster/dagster/_core/storage/cloud_storage_compute_log_manager.py

```diff
@@ -51,6 +51,21 @@ def delete_logs(
     def download_url_for_type(self, log_key: Sequence[str], io_type: ComputeIOType) -> str:
         """Calculates a download url given a log key and compute io type."""
 
+    def uri_or_path_for_type(self, log_key: Sequence[str], io_type: ComputeIOType) -> Optional[str]:
+        """Calculates a download uri given a log key and compute io type."""
+        return None
+
+    def get_log_manager_metadata(self) -> dict[str, str]:
+        """Returns metadata about the log manager."""
+        return {"type": self.__class__.__name__}
+
+    def get_serialized_log_manager_metadata(self) -> Optional[str]:
```

Comment:
our serdes library is applied recursively, so you shouldn't need to manually convert this to a string. The `get_log_manager_metadata` property should be enough.

Reply:
yes, see PR #27000 (not quite solved but that's the intention).